src/osd/OSD.cc

```diff
@@ -7647,6 +7647,11 @@ vector<DaemonHealthMetric> OSD::get_health_metrics()
     too_old -= cct->_conf.get_val<double>("osd_op_complaint_time");
     int slow = 0;
     TrackedOpRef oldest_op;
+    OSDMapRef osdmap = get_osdmap();
+    map<uint8_t, int> slow_op_types;
```

Comment:
descriptions?

Reply:
Added description for both slow_op_types and slow_op_pools